### PR TITLE
Add EIP: Dynamic Compliant Interop Security Token

### PR DESCRIPTION
**Pull request by abhinav-d3v**
Dynamic Compliant Interop Security Token(DyCIST) is a proposed token standard that extends ERC-1155 and combines its functionality with other existing token standards, introducing unique features for interoperability, cross-chain operation, and token wrapping.

### CODE REVIEW

**Review by bitcoinfinancier — APPROVED** [review]

File: EIPS/eip-7517.md [hunk 1]
```diff
@@ -0,0 +1,607 @@
+---
+eip: 7517
```
Comment (abcoathup) [inline EIPS/eip-7517.md:2]:
```suggestion
eip: 7518
```
Assigning next sequential EIP/ERC number.  7517 already issued to #7682 

Please also update the filename.

File: EIPS/eip-7517.md [hunk 2]
```diff
@@ -0,0 +1,607 @@
...
+title: Global Compliant STO & Interoperable Asset Lifecycle
+author: Abhinav (@abhinav-d3v),Prithvish Baidya(@d4mr), Rajat Kumar(@rajatwasan), Prasanth Kalangi <pkalangi@tassets.com>
+discussions-to:
```
Comment (abcoathup) [inline EIPS/eip-7517.md:5]: Please add a discussions post on Eth Magicians with a link to the ERC

File: EIPS/eip-7517.md [hunk 3]
```diff
@@ -0,0 +1,607 @@
+---
+eip: 7517
+title: Global Compliant STO & Interoperable Asset Lifecycle
```
Comment (abcoathup) [inline EIPS/eip-7517.md:3]:
```suggestion
title: Partitioned semi-fungible token
```
Suggest improving the title,

**Review by Mikhai56 — APPROVED** [review]

**Review by abhinav-d3v — COMMENTED** [review]
all necessary changes are done

**Comment by SamWilsn** [comment]
I am closing this pull request because we are in the process of separating EIPs and ERCs into distinct repositories. Unfortunately, as far as we are aware, GitHub does not provide any tools to ease this migration, so every pull request will need to be re-opened manually.

As this is a PR to create / modify an ERC, I will kindly ask you to redirect this to the new repository at [ethereum/ERCs](https://github.com/ethereum/ERCs). **We have prepared [a guide](https://github.com/ethereum/ERCs/issues/8) to help with the process.**

If there is relevant history here, please link to this PR from the new pull request.

On behalf of the EIP Editors, I apologize for this inconvenience.

**Pull request closed without merging**